### ISSUE TITLE
JAWS giving no feedback if a pop up is opened using the Links List Dialog

### ISSUE THREAD

**Issue by Sarthak-Kulkarni** [FreedomScientific/VFO-standards-support#623]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Go to https://codepen.io/sarthak-kulkarni305/pen/LYeomOb
> 2. Make sure that the focus is not on the link that opens pop-up.
> 3. Open Links List Dialog (Insert + F7)
> 4. Activate the link to open pop-up
> 5. JAWS gives no feedback

### Expected result

> What did you expect to happen?
JAWS should have read the pop-up header in feedback.

### Actual result
> What actually did happen?
JAWS gave no feedback.

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...
https://codepen.io/sarthak-kulkarni305/pen/LYeomOb

### Additional Information
JAWS gives proper feedback if the link is activated manually (using TAB or by click). The problem occurs when using the Links List Dialog provided by JAWS.

#### JAWS version and build number
JAWS 2021, build- 2021.2111.13
 
#### Operating System and version
Microsoft Windows [Version 10.0.18363.2212]

#### Browser and version:
Any
